code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -285,8 +285,6 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 	return 1
 
 /datum/gas_mixture/share(datum/gas_mixture/sharer, atmos_adjacent_turfs = 4)
-	if(!sharer)

```

Comment:
In testing i've had a `warning()` in this branch, it has never fired and my testing included a round i meteored, left standing and meteored again

Reply:
so we can expect to see it in the live logs within hours then :grin:

Reply:
yeah, it's a given